### PR TITLE
Rebuild Turkish Wikipedia's model based on 2020 labeling campaign

### PR DESCRIPTION
**Pull request by kevinbazira**
Bug: T257359

### CODE REVIEW

File: config/wikis/trwiki.yaml
Comment (halfak) [inline config/wikis/trwiki.yaml:36]: You'll want to merge this with `autolabeled_revisions.20k_2020`

So this block should look like this: 
```
merged_samples:
  labeled_revisions.20k_2015:
    - human_labeled_revisions.20k_2015
  labeled_revisions.20k_2020:
    - human_labeled_revisions.3k_2020
    - autolabeled_revisions.20k_2020
```

File: config/wikis/trwiki.yaml
Comment (halfak) [inline config/wikis/trwiki.yaml]: This will change to `labeled_revisions.20k_2020` and the large sample will be something like "40k_2015_2020"

File: model_info/trwiki.goodfaith.md
Comment (halfak) [inline model_info/trwiki.goodfaith.md]: Minor loss in fitness could be because of the data merging work I described above.  It might get cleaned up when the rest of the sample is merged in.

**Comment by halfak** [comment]
The travis failure is weird.  Seems to be python 3.5 only.  I'm guessing there's a weird version mismatch in there with some part of the test software (pytest, coverage) that doesn't support 3.5.  I bet the right solution is to pin a version of pytest or codecov.   I don't have time to look into it right now, but I'll try to find some time soon.  Feel free to beat me to it by running the command that failed `coverage run --source editquality -m pytest -m "not nottravis"` locally.

File: config/wikis/trwiki.yaml
Comment (kevinbazira) [reply config/wikis/trwiki.yaml:36]: Thank you Aaron, this has been fixed.

File: config/wikis/trwiki.yaml
Comment (kevinbazira) [reply config/wikis/trwiki.yaml]: Thank you Aaron, this has been fixed.

File: model_info/trwiki.goodfaith.md
Comment (halfak) [inline model_info/trwiki.goodfaith.md:75]: Improvement

File: model_info/trwiki.damaging.md
Comment (halfak) [inline model_info/trwiki.damaging.md:75]: Improvement

File: model_info/trwiki.goodfaith.md
Comment (halfak) [inline model_info/trwiki.goodfaith.md:21]: We can see the # of observations double

File: model_info/trwiki.damaging.md
Comment (halfak) [inline model_info/trwiki.damaging.md:21]: We can see the # of observations double

**Comment by halfak** [comment]
This looks good.  I'm running a test with travis now to deal with the failure you are seeing.  See https://github.com/wikimedia/editquality/pull/234.  If that builds successfully, you could merge it and then rebase this PR and that should fix the problem.  If the problem persists in #234, I'll need to dig further.

**Comment by elukey** [comment]
@kevinbazira can you rebase and re-submit? I just merged Aaron's change for pytest, after that Travis should finally work.

**Review by halfak — APPROVED** [review]
Looks good.  Thanks!

**Comment by halfak** [comment]
@elukey, is it OK if I merge PRs like this after reviewing/approving or would you like to have someone from the ML team make the final call re. merging?

**Comment by elukey** [comment]
> @elukey, is it OK if I merge PRs like this after reviewing/approving or would you like to have someone from the ML team make the final call re. merging?

If possible I'd ask to wait for people to merge :)

**Review by accraze — APPROVED** [review]
LGTM

**Pull request merged**
